modules/nodes-nodes-rebooting-affinity.adoc

```diff
@@ -50,4 +50,4 @@ This example assumes the container image registry pod has a label of
 expression.
 
 . Enable the `*MatchInterPodAffinity*` scheduler predicate in the scheduling policy file. 
-For more information, see xref:../../nodes/scheduling/nodes-scheduler-default.adoc#nodes-scheduler-default[Configuring the Default Scheduler].
+For more information, see Configuring the Default Scheduler.
```

Comment:
I don't think this is a good idea to leave a non-xref dependency like this (specifying the title but it not being an xref). Because even though it's not technically dependent, it doesn't make sense w/o the other module. I don't know how others have been handling it, but maybe you could say something like "For more information, see the documentation on configuring the default scheduler"? I don't know, wdyt?

Reply:
That topic doesn't belong in here! Will remove and consider your question in context!